gdb-xtensa-esp32-elf/build.sh

```diff
@@ -21,6 +56,9 @@ cd binutils-gdb-esp32-src
     --with-expat \
     --with-libexpat-prefix="$PREFIX" \
     --with-libiconv-prefix="$PREFIX" \
+    --with-libexpat-type=static \
```

Comment:
For the linux build at least, this reduces the runtime deps a bit, though I didn't update the `run` requirements:
```diff--- /proc/self/fd/13    2022-08-03 11:04:13.584491481 -0400
+++ /proc/self/fd/15    2022-08-03 11:04:13.584491481 -0400
@@ -1,8 +1,6 @@
 /lib64/ld-linux-x86-64.so.2
 libc.so.6
 libdl.so.2
-libexpat.so.1
-libgcc_s.so.1
 libgmp.so.10
 libiconv.so.2
 liblzma.so.5
@@ -12,7 +10,6 @@
 libpthread.so.0
 libpython3.8.so.1.0
 librt.so.1
-libstdc++.so.6
 libtinfow.so.6
 libutil.so.1
 libz.so.1
```